src/app/app.component.ts

```diff
@@ -26,16 +26,12 @@ export class AppComponent implements OnInit, OnDestroy {
   userinfo: Userinfo;
   userImageDoc: HalDoc;
 
-  routerSub: Subscription;
-  routerState: RouterModel;
-
   constructor(
     public store: Store<any>,
     private angulartics2GoogleAnalytics: Angulartics2GoogleAnalytics,
     private user: UserinfoService
   ) {
     this.store.dispatch(new ACTIONS.CmsAccountAction());
```

Comment:
left this cms account stuff alone in the app.component because eventually will use to just using id and can move initialization to effects at that time